client/api.c

```diff
@@ -281,6 +281,12 @@ TDNFCheckLocalPackages(
     pr_info("Checking all packages from: %s\n", pszLocalPath);
 
     pCmdLinePool = pool_create();
+
+    if (pool_setdisttype(pCmdLinePool, DISTTYPE_RPM) < 0) {
```

Comment:
I should probably put this with `pool_create()` into a separate function because it repeats 4 times in the code.

Reply:
Done.